### ISSUE TITLE
508-defect-2 [PDF]: PDF should be remediated to pass accessibility checks

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#24685]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## Summary

The checker found problems which may prevent the document from being fully accessible.

- Needs manual check: 2
- Passed manually: 0
- Failed manually: 0
- Skipped: 1
- Passed: 12
- Failed: 17

### Document
- Tagged PDF | Failed | Document is tagged PDF
- Primary language | Failed | Text language is specified
- Title | Failed | Document title is showing in title bar

### Page Content
- Tagged content | Failed | All page content is tagged
- Tab order | Failed | Tab order is consistent with structure order

### Alternate Text
- Figures alternate text | Failed | Figures require alternate text
- Nested alternate text | Failed | Alternate text that will never be read
- Associated with content | Failed | Alternate text must be associated with some content
- Hides annotation | Failed | Alternate text should not hide annotation
- Other elements alternate text | Failed | Other elements that require alternate text

### Tables
- Rows | Failed | TR must be a child of Table, THead, TBody, or TFoot
- TH and TD | Failed | TH and TD must be children of TR
- Headers | Failed | Tables should have headers
- Regularity | Failed | Tables must contain the same number of columns in each row and rows in each column

### Lists
- List items | Failed | LI must be a child of L
- Lbl and LBody | Failed | Lbl and LBody must be children of LI

### Headings
- Appropriate nesting | Failed | Appropriate nesting

## Remediation
[These issues can be remediated by following guidance from Adobe.](https://helpx.adobe.com/acrobat/using/create-verify-pdf-accessibility.html)

## Acceptance Criteria

- [ ] PDF passes accessibility checks

## Steps to Recreate

TBD

**Comment by jparker0425** [comment]
This will need to be approached in  two step process: 

1. #25081
2. Need to meet with Platform about the broader scope of this due to it being a widespread issue. 
3. @dillo to bring up in VSA/VSP meeting to determine priority.
